content/en/docs/topics/advanced.md

```diff
@@ -143,3 +143,68 @@ func main() {
 }
 
 ```
+
+## Storage backends
+
+By default, Helm stores release information in Secrets in the namespace of
+the release.
+
+### ConfigMap storage backend
+
+To enable the ConfigMap backend, you'll need to set the environmental variable
+`HELM_DRIVER` to `configmap`.
+
+You can set it in a shell as follows:
+
+```shell
+export HELM_DRIVER=configmap
+```
+
+If you want to switch from the default backend to the ConfigMap
+backend, you'll have to do the migration for this on your own. You can retrieve
+release information with the following command:
+
+```shell
+kubectl get secret --all-namespaces -l "owner=helm"
+```
+
+**PRODUCTION NOTES**: It is not recommended to use ConfigMaps for sensitive data as
+it is stored unencoded. Secrets data is stored base64-encoded.
```

Comment:
Anyone who cares about sensitive information storage is going to see the "base64-encoded" and know that's not secure. We don't want them to think we think it's secure. I would add something to the end like...

> ... and secrets can be configured for encrypted storage.

The last part I'd link to https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/

Reply:
Thats a good point. Added the clarification.